### PR TITLE
feat: Skip setup tag after first setup.

### PR DESCRIPTION
**Pull request by codefriar**
Team, 

This PR enables us to conditionally disable the setup and tor tags. 

The utils.rb file has been updated to look for a config_dir/.setup file. If present, the setup tasks are skipped. 
Additionally, if tor is not enabled, the tor tasks are entirely skipped.

This does dramatically speed up deployments. 

@denis-ev  - if we tag each service as a role, would tags allow us to get around the whole deprecated warning bit?

### CODE REVIEW

**Comment by codefriar** [comment]
closes !260

**Comment by codefriar** [comment]
also closes #259

File: roles/vivumlab_deploy/tasks/main.yml [hunk 1]
```diff
@@ -201,25 +213,21 @@
...
   when: "lookup('vars', 'enable_tor')"
   become: yes
-
-- name: Get UID
-  command: id -u {{ vlab_ssh_user }}
-  register: uid_output
-
-- name: Get GID
-  command: id -g {{ vlab_ssh_user }}
-  register: gid_output
```
Comment (denis-ev) [inline roles/vivumlab_deploy/tasks/main.yml:217]: Why did you remove this? 
I'm quite sure we use it everywhere.

**Review by denis-ev — APPROVED** [review]
